mautic/mautic
# Add test for UI installer to ensure basic config works (5.x)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

There is currently no test for the UI installer, which is a non-ideal situation :)
in #11142 and #11281 changes were added that did break the UI installer.

This PR addresses this by
* adding a basic test
* providing a fix for the regressing in  #11281

The test itself is based upon #11144 (thanks @escopecz!)

It's really weird people are able to test the 5.x branch UI installer locally or in gitpod, as it was broken in 2 places.



#### Steps to test this PR:


1. check the outcome of the tests
2. install Mautic via the UI locally to verify the fix for #11281
